tls/s2n_recv.c

```diff
@@ -130,7 +130,6 @@ ssize_t s2n_recv(struct s2n_connection * conn, void *buf, ssize_t size, s2n_bloc
     struct s2n_blob out = {.data = (uint8_t *) buf };
 
     if (conn->closed) {
-        GUARD(s2n_connection_wipe(conn));
```

Comment:
these changes to s2n_connection_wipe look unrelated. should they be pulled out and merged separately?